feat(applications-monitoring-centreon-dem-restapi) and deprecate(applications-monitoring-quanta-restapi) CTOR-2076 (#4916)
centreon/centreon-documentation: pp/integrations/plugin-packs/procedures/applications-monitoring-centreon-dem-restapi.md

|:------------------------|:------------------------------------------------------------------|:-----------------------------------------|:---------:|
| User Journey Incidents  | App-Monitoring-Centreon-DEM-Userjourney-Incidents-Restapi-custom  | Check incidents for a given user journey | X         |
| User Journey Statistics | App-Monitoring-Centreon-DEM-Userjourney-Statistics-Restapi-custom | Check statistics for a user journey      | X         |

> The services listed above are not created automatically when a host template is applied. To use them, [create a service manually](/docs/monitoring/basic-objects/services), then apply the service template you want.

> If **Discovery** is checked, it means a service discovery rule exists for this service template.

</TabItem>
</Tabs>

### Discovery rules

#### Service discovery

| Rule name                                                   | Description                                            |
|:------------------------------------------------------------|:-------------------------------------------------------|

> | App-Monitoring-Centreon-DEM-Restapi-Userjourneys-Incidents  | Discover user journeys of a site to monitor incidents  |

Reviewer: ```suggestion
| App-Monitoring-Centreon-DEM-Restapi-Userjourneys-Incidents  | Discover user journeys of a site and monitor incidents  |
```

> | App-Monitoring-Centreon-DEM-Restapi-Userjourneys-Statistics | Discover user journeys on a site to monitor statistics |

Reviewer: ```suggestion
| App-Monitoring-Centreon-DEM-Restapi-Userjourneys-Statistics | Discover user journeys on a site and monitor statistics |
```

|:-----------------------------------|:------|
| centreon.dem.incidents.total.count | count |
| incident-status                    | N/A   |
| incident-type                      | N/A   |
| incident-duration                  | N/A   |

</TabItem>
<TabItem value="User Journey Statistics" label="User Journey Statistics">

| Name                                           | Unit |
|:-----------------------------------------------|:-----|
| journey-performance-score                      | N/A  |
| *metrics*#journey.herotime.milliseconds        | ms   |
| *metrics*#journey.speedindex.time.milliseconds | ms   |
| *metrics*#journey.ttfb.milliseconds            | ms   |
| interaction-performance-score                  | N/A  |
| *metrics*#herotime.milliseconds                | ms   |
| *metrics*#speedindex.time.milliseconds         | ms   |
| *metrics*#ttfb.milliseconds                    | ms   |

</TabItem>
</Tabs>

## Prerequisites

In order to monitor Quanta, you must have [a valid API token](/dem/installation/monitor-production-events#authentication-and-token-generation), and [have user journeys configured in Quanta](/dem/configuration/user-journey/create-a-scenario).

> Data from Real User Monitoring will only be accessible if you have completed [the integration of Quanta's RUM TAG](/dem/installation/real-user-monitoring-installation#find-the-tag-to-insert-on-my-site).

Author: ```suggestion
In order to monitor Centreon DEM, you must have [a valid API token](/dem/installation/monitor-production-events#authentication-and-token-generation), and [have user journeys configured in Centreon DEM](/dem/configuration/user-journey/create-a-scenario).
Data from Real User Monitoring will only be accessible if you have completed [the integration of Centreon DEM's RUM TAG](/dem/installation/real-user-monitoring-installation#find-the-tag-to-insert-on-my-site).
```

> Data from Real User Monitoring will only be accessible if you have completed [the integration of Centreon DEM's RUM TAG](/dem/installation/real-user-monitoring-installation#find-the-tag-to-insert-on-my-site).

Reviewer: ```suggestion
Data from Real User Monitoring will only be accessible if you have completed [the integration of Centreon DEM's RUM tag](/dem/installation/real-user-monitoring-installation#find-the-tag-to-insert-on-my-site).
```
